api/docs/docs/data.md

```diff
@@ -0,0 +1,210 @@
+---
+id: data
+title: Data Definitions
+---
+
+Read more about the data included in the Covid Act Now API.
+
+
+## Cases
+
+### Cases
+
+  Cumulative confirmed or suspected cases.
+
+**Where to access**  
+* CSV column names: ``actuals.cases``
+* JSON file fields: ``actuals.cases``, ``actualsTimeseries.*.cases``
+
+### New Cases
+
+  New confirmed or suspected cases.
+
+New cases are a processed timeseries of cases - summing new cases may not equal
+the cumulative case count.
+
+Processing steps:
+ 1. If a region does not report cases for a period of time, the first day
+    cases start reporting again will not be included. This day likely includes
+    multiple days worth of cases and can be misleading to the overall series.
+ 2. Any days with negative new cases are removed.
+ 3. An outlier detection filter is applied to the timeseries, removing any data points that
+    seem improbable given recent case numbers.  Many times this is due to a backfill of
+    previously unreported cases.
```

Comment:
Alt: "We apply an outlier detection filter to the timeseries, which removes any data points that seem improbable given recent numbers. Many times this is due to backfill of previously unreported cases."